src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/SymmetricAlgorithm.cs

```diff
@@ -125,6 +125,7 @@ public virtual CipherMode Mode
 
             set
             {
+                // Review note: should we throw PNSE instead for OFB and CFB?
```

Comment:
Though note that CTS seems to have already resulted in this exception in Desktop.